lib/rubygems/install_update_options.rb

```diff
@@ -181,10 +181,19 @@ def add_install_update_options
   end
 
   ##
-  # Default options for the gem install command.
+  # Default options for the gem install and update commands.
+
+  def install_update_options
+    {
+      :document => %w[ri],
+    }
+  end
+
+  ##
+  # Default description for the gem install and update commands.
 
   def install_update_defaults_str
-    '--document=rdoc,ri --wrappers'
+    '--document=ri'
```

Comment:
Do you know what this `--wrappers` thing is?

Reply:
The default value of `--wrappers` is `true` on `DependencyInstaller`. I remove it with the other options.

Reply:
Yeah, I understood, I was just asking if you now what it does. I think it's about creating binstubs wrappers instead of symlinks, but not sure. Anyways, I was just curious.